paging/src/test/java/com/hannesdorfmann/adapterdelegates4/paging/PagedListDelegationAdapterTest.java

```diff
@@ -49,7 +49,7 @@ public int getItemCount() {
     @Test
     public void adapterDelegateManagerIsNull() {
         try {
-            AsyncListDifferDelegationAdapter<Object> adapter = new AsyncListDifferDelegationAdapter<Object>(callback, null) {
+            PagedListDelegationAdapter<Object> adapter = new PagedListDelegationAdapter<Object>(null, callback) {
```

Comment:
I think this was a copy and paste error and this change made it visible.

Reply:
Oh, nice, thank you